dist/profile/manifests/kubernetes/resources/datadog.pp

```diff
@@ -0,0 +1,62 @@
+# Deploy datadog resources on kubernetes cluster
+class profile::kubernetes::resources::datadog (
```

Comment:
I don't think that datadog class should move to a definition, but we can clearly 'extract' a part of it. 
which configure and apply kubernetes resource